views/apartments/new.erb

```diff
@@ -0,0 +1,10 @@
+<div class="addForm">
+<form method="post" action="/apartments">
+  <label for="address">Address:</label> <input id="address" type="text" name = "address">
```

Comment:
maybe the creation is not working because of the space between your `name` attribute and equals sign?